grouper/models/public_key.py

```diff
@@ -2,6 +2,63 @@
 from sqlalchemy import Column, Integer, String, Text, ForeignKey, DateTime
 from sqlalchemy.orm import relationship
 from grouper.models.base.model_base import Model
+from grouper.models.public_key_tag_map import PublicKeyTagMap
+
+
+def _perm_list_to_dict(perms):
+    # type: List[Permission] -> Dict[str, Dict[str, Permission]]
```

Comment:
i believe this will fail `mypy` since
1. Permission is not imported
2. and what's returned is actually a `sqlalchemy.engine.RowProxy` more like a Permission Tuple. :/